Library/Formula/node.rb

```diff
@@ -25,6 +25,7 @@ class Node < Formula
   option "without-completion", "npm bash completion will not be installed"
 
   depends_on :python => :build
+  depends_on :xcode => :build
```

Comment:
Oh. Nobody seems to have told the ` npm ` group this :wink:. I'll remove it, but if everyone who has Xcode installed can install this fine, and those who only have the CLT package installed can't, without much obvious difference aside that fact between machines, I'm not really sure what else it could be as a root cause :confounded:.